recipe/meta.yaml

```
@@ -32,43 +32,131 @@ requirements:
     - apache-beam >=2.12 # extra:beam
     - bottle >=0.12.13 # extra:bottle
     - celery >=3 # extra:celery
+    - celery-redbeat >=2 # extra:celery-redbeat
     - chalice >=1.16.0 # extra:chalice
     - clickhouse-driver >=0.2.0 # extra:clickhouse-driver
     - django >=1.8 # extra:django
     - falcon >=1.4 # extra:falcon
     - fastapi >=0.79.0 # extra:fastapi
     - flask >=0.11 # extra:flask
     - blinker >=1.1 # extra:flask
+    - markupsafe # extra:flask
```

```suggestion
    #- markupsafe # extra:flask
```
It's not needed without pinning